mysql-test/suite/rocksdb.rpl/t/rpl_rocksdb_stm_mixed_crash_safe.test

```diff
@@ -0,0 +1,18 @@
+--source include/big_test.inc
+--source include/master-slave.inc
```

Comment:
All of these were copied from elsewhere in the mtr suite and switched to RocksDB, I tried to catch and fixup some of this kind of stuff as I went along and obviously missed some in my haste :-\